app/assets/stylesheets/components/_header-notice.scss

```diff
@@ -0,0 +1,30 @@
+.app-c-header-notice {
+  margin-bottom: govuk-spacing(8);
+}
+
+.app-c-header-notice__header {
+  background-color: govuk-colour("black");
+  padding: govuk-spacing(4);
+}
+
+.app-c-header-notice__title {
+  color: govuk-colour("white");
+  margin: 0;
+}
+
+.app-c-header-notice__content {
+  border: 5px solid govuk-colour("black");
+  padding: govuk-spacing(4);
+}
+
+.app-c-header-notice__list {
+  margin-bottom: 0;
+}
+
+.app-c-header-notice__link-intro {
+  margin-bottom: govuk-spacing(1);
+}
+
+.app-c-header-notice__link {
```

Comment:
similarly, we can `@extend %govuk-link;` here and remove the class used in markup